app/views/obituaries/covid/index.html.erb

```diff
@@ -1,39 +1,27 @@
 <div class="container">
-<h1 class="my-4">COVID-19 Obituaries</h1>
-<% @obituaries.each do |obit| %>
-  <div class="row" style= "background-color:white;">
-    <div class="col-md-3 p-4">
-      <%= link_to image_tag(obit.image_url), "/obituaries/#{obit.id}" %>
-    </div>
-    <div class="col-md-9 p-4">
-      <h3><%= "#{obit.first_name} #{obit.last_name}" %></h3>
-      <p><%= obit.description %></p>
-      <a class="btn btn-dark" href= <%= "/obituaries/#{obit.id}" %>>View Obituary</a>
-    </div>
+  <h1 class="my-4" id="results-header">COVID Related Community Obituaries:</h1>
+  <input type="button" id="change" value="View NYTimes Results" class="btn btn-outline-secondary mr-1">
+  <div id="community-results">
+    <%= render partial: "browsing_index" %> 
+  </div>
+  <div id="nytimes-results" class="hide-dom">
+    <%= render partial: "nytimes_index" %>
   </div>
-  <hr>
-<% end %>
-
-  <% @obituaries_api.each do |obit| %>
-    <section class="covid_obituary">
-      <div class="headline">
-        <%= obit.headline %>
-      </div>
-      <div class="abstract">
-        <%= obit.abstract %>
-      </div>
-      <div class="web_url">
-        <%= obit.web_url %>
-      </div>
-      <div class="pub_date">
-        <%= obit.pub_date %>
-      </div>
-      <div class="word_count">
-        <%= obit.word_count %>
-      </div>
-      <div class="image">
-        <%= obit.image %>
-      </div>
-    </section>
-  <% end %>
 </div>
+
+<script type="text/javascript">
+  $('#change').click(function() {
+     $('#community-results, #nytimes-results').toggle();
+    $(this).prop('value') == 'View NYTimes Results' ? communityText() : nytimesText();
+  });
```

Comment:
This changes the text on the button, the header, and the partial being rendered when it is clicked.

Reply:
Nice work here!

Reply:
Javascript is magic.